hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeUsage.java

```diff
@@ -185,7 +185,7 @@ public static boolean hasVolumeEnoughSpace(long volumeAvailableSpace,
                                              long requiredSpace,
                                              long volumeFreeSpaceToSpare) {
     return (volumeAvailableSpace - volumeCommittedBytesCount) >
-        Math.max(requiredSpace, volumeFreeSpaceToSpare);
```

Comment:
@peterxcli , thanks for working on this.  I would prefer following statement which is more straight forward. 

return (volumeAvailableSpace - volumeCommittedBytesCount - volumeFreeSpaceToSpare) >  requiredSpace;

Besides, could you help to refactor the HddsDispatcher#isVolumeFull in this patch too, make isVolumeFull call hasVolumeEnoughSpace, so that we will have single central place to control how the disk full is detected.